public/app/core/services/util_srv.ts

```diff
@@ -52,18 +52,14 @@ export class UtilSrv {
   showConfirmModal(payload) {
     const scope = this.$rootScope.$new();
 
-    scope.onConfirm = () => {
-      payload.onConfirm();
-      scope.dismiss();
-    };
-
     scope.updateConfirmText = value => {
       scope.confirmTextValid = payload.confirmText.toLowerCase() === value.toLowerCase();
     };
 
     scope.title = payload.title;
     scope.text = payload.text;
     scope.text2 = payload.text2;
+    scope.text2htmlBind = payload.text2htmlBind;
```

Comment:
Allows for using html in the text string. Used to show link to docs in the delete confirmation.